web/components/NavBlock/NavBlock.module.css

```diff
@@ -2,6 +2,21 @@
   margin: 80px 0;
 }
 
+.navEnter .item,
```

Comment:
Nit: I think it would be slightly clearer if the "base" class of the nav is also used, to help show where it comes from. (Drawing a parallel to `.block--modifier` in BEM.) I.e. `.nav.navEnter .item`, or maybe better with a class rename, something like `.nav.isIntersecting .item`

Also, could these declaration blocks be moved below the existing `.item, .fakeItem` block? They're roughly ordered by which elements are targeted, in the same order the elements (first) appear in the DOM/JSX, with the "base" styling first and variations after. First for base styling, then again within each `@media` block from narrowest to widest. (Sorry this convention is undocumented.)

Reply:
Why aren't we using BEM anyway? Then it wouldn't have a specificity war going on in the first place.

Also, if we where to use SASS I think it would be much easier to attach @media blocks, :hovers, etc to the actual classnames making the code much clearer and easier to read.

(But I'll update the classnames so it matches the rest of the css).

Reply:
There is no specificity war, just a minor readability nit.

We aren't using BEM because `css-modules` already does the same "namespacing" job, IMO.

(To be clear, any comment we prefix "Nit:" is not regarded as "blocking" the PR approval. Maybe that was already obvious, I don't know how widespread the practice is.)